Assets/UI/Panels/citypanel.lua

```diff
@@ -1188,6 +1268,47 @@ end
 function OnShutdown()
     -- Cache values for hotloading...
     LuaEvents.GameDebug_AddValue("CityPanel", "isHidden", ContextPtr:IsHidden() );
+
+        -- Game Core Events
```

Comment:
these "Remove" calls appear in the unmodified version.  Specific CQUI functions called out.